src/Commands/RefreshLQIP.php

```diff
@@ -66,14 +67,24 @@ public function handle()
 
                     $crop_params = Arr::only((array) $attached_media, $this->cropParamsKeys);
 
+                    $imageService = config('twill.media_library.image_service');
+
                     $url = ImageService::getLQIPUrl($uuid, $crop_params + ['w' => $lqip_width]);
 
+                    if ($imageService === Glide::class) {
+                        // Glide is a local imaging service so we should make sure we also utilize the local url.
+                        // This solution might not work at all in some cases.
+                        // I feel like this is not optimal and we could maybe directly use the glide api in this case
+                        // but maybe that is not worth the effort (for now).
+                        $url = url($url);
+                    }
```

Comment:
Not sure this is necessary. While trying to get Glide working locally I found I just needed to set the following env variables locally (Worth noting I also had previously run `php artisan storage:link`):

```
GLIDE_BASE_URL=http://www.localhost
GLIDE_BASE_PATH=/storage
```

Perhaps this just needs documenting?